django/db/backends/utils.py

```diff
@@ -61,7 +61,7 @@ def callproc(self, procname, params=None, kparams=None):
                 "Keyword parameters for callproc are not supported on this "
                 "database backend."
             )
-        if not apps.ready:
+        if not apps.ready and not apps.stored_app_configs:
```

Comment:
Do we want to store these in a property eg `Apps.is_initializing` where we can also put a comment/docstring that clarifies what Simon explained about `stored_app_configs` is mostly tied to `@override_settings` ? 🤔

Reply:
I don't think it's worth a new attribute.

> where we can also put a comment/docstring that clarifies what Simon explained about

Yes, please.